src/briefcase/commands/create.py

```diff
@@ -302,6 +302,8 @@ def install_app_support_package(self, app: BaseConfig):
         except KeyError:
             self.logger.info("No support package required.")
         else:
+            if update:
```

Comment:
Does this need an `and support_path.exists() and support_path.is_dir()` qualifier? It's not a likely edge case, but it's conceptually possible.